extensions/skins/conversation_v1/Conversation.js

```diff
@@ -730,6 +651,7 @@ oppia.directive('progressDots', [function() {
       });
 
       $scope.changeActiveDot = function(index) {
+        $scope.progressDotClicked = true;
```

Comment:
Oh, wait -- this is an animation that's meant to be handled outside the directive. So in that case I think we should keep track of it entirely outside progressDots. Couldn't this be done with currentProgressDotIndex and numProgressDots?